test/LifecycleServiceTest.js

```diff
@@ -57,28 +56,6 @@ describe('LifecycleService', function () {
         });
     });
 
-    it('client should emit starting, started, shuttingDown and shutdown events in order (via import config)', function (done) {
```

Comment:
Is there an alternative to this test somewhere else? Why is it removed?

Reply:
Loading lifecycle listeners from the config file test is covered in `ConfigBuilderTest#listenerConfig` and emitting lifecycle events test is covered `LifecycleServiceTest#client should emit starting, started, shuttingDown and shutdown events in order`.